ssd1306/ssd1306.go

```diff
@@ -338,3 +343,22 @@ func (b *SPIBus) tx(data []byte, isCommand bool) error {
 func (d *Device) Size() (w, h int16) {
 	return d.width, d.height
 }
+
+// DrawBitmap copies the bitmap to the screen at the given coordinates.
+func (d *Device) DrawBitmap(x, y int16, bitmap pixel.Image[pixel.Monochrome]) error {
+	width, height := bitmap.Size()
+	if x < 0 || x+int16(width) > d.width || y < 0 || y+int16(height) > d.height {
+		return errOutOfRange
+	}
+
+	zeroColor := color.RGBA{0, 0, 0, 255}
+	for i := 0; i < width; i++ {
+		for j := 0; j < height; j++ {
+			if bitmap.Get(i, j).RGBA() != zeroColor {
+				d.SetPixel(x+int16(i), y+int16(j), color.RGBA{255, 255, 255, 255})
+			}
```

Comment:
You are only changing dark pixels to light pixels, but make it impossible to change light pixels to dark? Why is that?

This makes a lot more sense to me:

```suggestion
			d.SetPixel(x+int16(i), y+int16(j), bitmap.Get(i, j).RGBA())
```

Reply:
That looks a lot better @aykevl thanks. Change made.